src/main/java/core/basesyntax/service/impl/OperationHandlerImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationService;
+
+public class OperationHandlerImpl implements OperationService {
+    @Override
+    public Operation getOperation(String operationType) {
```

Comment:
It's not very flexible solution to use switch case here. What if you would need to support 100 operations? Simply iterate through all operation values and return if values match.

Also, it's redundant to create separate service to get Operation by it's value, you can do it in Operation itself.